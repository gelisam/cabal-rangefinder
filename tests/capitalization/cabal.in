Name:                hello
Version:             0.1.0.0
Build-Type:          Simple
Cabal-Version:       >=1.10

Library
  Build-Depends:       base >= 4.6,
                       containers,

                       mtl
  Exposed-Modules:     Hello
