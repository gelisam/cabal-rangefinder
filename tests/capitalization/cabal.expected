Name:                hello
Version:             0.1.0.0
Build-Type:          Simple
Cabal-Version:       >=1.10

Library
  Build-Depends:       base >=4.6.0.1,
                       containers >=0.4.2.1,

                       mtl >=2.1.2
  Exposed-Modules:     Hello
